.evergreen/config.yml

```diff
@@ -174,7 +174,7 @@ functions:
       params:
         script: |
           ${PREPARE_SHELL}
-          MONGODB_VERSION=${VERSION} TOPOLOGY=${TOPOLOGY} AUTH=${AUTH} SSL=${SSL} STORAGE_ENGINE=${STORAGE_ENGINE} sh ${DRIVERS_TOOLS}/.evergreen/run-orchestration.sh
+          MONGODB_VERSION=${VERSION} ORCHESTRATION_FILE=${ORCHESTRATION_FILE} TOPOLOGY=${TOPOLOGY} AUTH=${AUTH} SSL=${SSL} STORAGE_ENGINE=${STORAGE_ENGINE} sh ${PROJECT_DIRECTORY}/.evergreen/run-orchestration.sh
```

Comment:
Using our own file allows us to add logic to override the used file. Passing `ORCHESTRATION_FILE` as well allows configurations to select a specific file instead of relying on the filename being assembled by the logic included.

Reply:
Was this borrowed from PHPC or is it new for PHPLIB (in which case it may be something you'd like to add back to PHPC)?

Reply:
I didn't find `ORCHESTRATION_FILE` in https://github.com/mongodb/mongo-php-driver/blob/master/.evergreen/config.yml, so I assume this is new. Feel free to create a ticket if you think it's worth porting this over to PHPC down the line.

Reply:
Created PHPC-1676 to track this.